tests/jeos/firstrun.pm

```diff
@@ -80,7 +80,7 @@ sub run {
         my $name = $con->name;
         enter_cmd("virsh console --devname console0 --force $name");
         # long timeout bsc#1210429
-        $initial_screen_timeout = 420;
+        $initial_screen_timeout = 420 if is_sle_micro;
```

Comment:
Ah, this long timeout was due to ignition/combustion. Then, let's change the comment to `# long timeout due to ignition/combustion bsc#1210429` :D